plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduBucketFunction.java

```diff
@@ -69,6 +69,11 @@ public int getBucket(Page page, int position)
                 .map(hashBucketSchema -> this.calculateSchemaLevelBucketId(page, partialRow, hashBucketSchema, position))
                 .collect(toImmutableList());
 
+        return getBucket(bucketIds, hashBucketSchemas);
```

Comment:
Can we rename the commit message to `Avoid using range partitions for Kudu bucketing `